index.html

```diff
@@ -153,28 +153,48 @@ <h1>Introduction</h1>
   <section>
     <h1>The Registration Process</h1>
     <p>
-Software implementers may find that the existing Decentralized Identifier
-Core specification [[DID-CORE]] is not suitable for their use case and
-might need to add a new parameters, properties, or values to this registry
-in order to achieve their use case in a globally interoperable fashion.
-In order to add a new parameter, property, or value to this registry, an
-implementer MUST:
-    </p>
-
-    <p class="issue">
-The DID WG is currently determining the process for managing entries for this
-registry.
+Software implementers might find that the existing Decentralized Identifier Core
+specification [[DID-CORE]] is not entirely capable of addressing their use case
+and might need to add a new parameters, properties, or values to this registry
```

Comment:
Do we need this minor change?

```suggestion
and might need to add a new parameters, properties, or values to these registries
```